apps/server/src/modules/board/service/content-element.service.ts

```diff
@@ -41,7 +41,7 @@ export class ContentElementService {
 	}
 
 	async create(parent: Card | SubmissionItem, type: ContentElementType): Promise<AnyContentElementDo> {
-		const element = this.contentElementFactory.build(type);
+		const element = this.contentElementFactory.build(type, parent.id);
```

Comment:
why is is the parentId needed here?

Reply:
For mainaining connection between tldraw data as board course node and data as websocket ydoc tldraw data - we are using parent.id (cardId) as docName/drawingName. So with creating course DrawingElement we are assigning to param drawingName = parent.id

Reply:
why dont you use the id of the element as parentId? Thats how files are handled too, and it would allow for multiple drawings in the same card

Reply:
and it would avoid element specific code here.

Reply:
Because as we got requirements - there should be one board per card